maxim/logger/livekit/store.py

```
@@ -211,3 +221,37 @@ def close_session(self, session_info: SessionStoreEntry):
 def get_session_store():
     """Get the global session store instance."""
     return _session_store
+
+class TTSStore:
+    """ 
+    Adding an additional store for TTS audio data.
+    This is done as TTS does not have a session reference.
+    """
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)